packages/hooks/src/usePlatformDemoAccount.ts

```diff
@@ -0,0 +1,13 @@
+import { useStore } from '@deriv/stores';
+
+const usePlatformDemoAccount = () => {
+    const { client } = useStore();
+    const { accounts } = client;
+    const account_list = Object.keys(accounts).map(loginid => accounts[loginid]);
+
+    const platform_demo_account = account_list.find(account => account.is_virtual);
```

Comment:
instead of adding it to this constant and return the constant you can directly return the calculated value.